src/_frameless.scss

```diff
@@ -48,20 +48,20 @@ $mobile: 480px;
 *          ... intermediate-and-smaller |
 */
 
-$small: "only screen and (max-width : #{$mobile}-1)";
-$medium: "only screen and (min-width : #{$mobile}) and (max-width : #{$tabletPortrait}-1)";
-$intermediate: "only screen and (min-width : #{$tabletPortrait}) and (max-width : #{$desktop}-1)";
+$small: "only screen and (max-width : #{$mobile - 1})";
+$medium: "only screen and (min-width : #{$mobile}) and (max-width : #{$tabletPortrait - 1})";
+$intermediate: "only screen and (min-width : #{$tabletPortrait}) and (max-width : #{$desktop - 1})";
```

Comment:
E.g. this was outputting `only screen and (max-width : 480px-1)` but now it outputs `only screen and (max-width : 479px)`